bitcoin/chainparams.h

```diff
@@ -22,6 +22,14 @@ struct chainparams {
 	const char *bip70_name;
 	const struct bitcoin_blkid genesis_blockhash;
 	const int rpc_port;
+	/**
+	 * BOLT 1: The default TCP port depends on the network used. The most common networks are:
```

Comment:
Hmm, our bolt quote detector is really dumb.  Needs just "BOLT #1:" on a line by itself, rest underneath.

Reply:
Once you do this, I can update BOLTVERSION to c74a3bbcf890799d343c62cb05fcbcdc952a1cf3, or you can do that in a separate commit.

You might also need https://github.com/ElementsProject/lightning/pull/5324